dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/MergeOfStoreNode.java

```diff
@@ -0,0 +1,58 @@
+package org.checkerframework.dataflow.cfg.node;
+
+import com.sun.source.tree.ExpressionTree;
+import com.sun.source.tree.Tree;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.javacutil.TreeUtils;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Objects;
+
+/**
+ * A node for the purpose of merging store in dataflow analysis. It usually follows an expression
+ * statement.
+ *
+ * <p>Note: Does not represent any AST structure.
```

Comment:
Alternatively, we could name it `ExpressionStatementNode` and then it is up to each analysis what they do if it.
At the moment the node name fixes what interpretation we have now, but maybe that could be different for other analyses?